.changeset/kind-apples-itch.md

```diff
@@ -0,0 +1,21 @@
+---
+'eslint-plugin-prettier': minor
+---
+
+feat: support ESLint flat config system
+
+`eslint-plugin-prettier` can be used in the flat config format like this with CJS:
+
+```js
+const prettier = require('eslint-plugin-prettier');
+
+module.exports = [prettier.configs['recommended-flat']];
```

Comment:
```suggestion
module.exports = [prettier.config.flatRecommended];
```

I still recommend this config name.

Reply:
But even better with a separate `flat.js` file, right?

Reply:
I don't have such preference, a clear document is enough IMO.